contracts/adder/deploy.sh

```diff
@@ -3,13 +3,20 @@
 NODE_URL="${NODE_URL:-ws://localhost:9944}"
 AUTHORITY="${AUTHORITY:-//Alice}"
 
-cargo contract build --release --quiet 1>&2
-cargo contract upload --url "$NODE_URL" --suri "$AUTHORITY" --quiet 1>&2
+function ink-build() {
+  docker run \
+    --network host \
+    -v ${PWD}:/code \
+    --platform linux/amd64 \
+    --rm -it public.ecr.aws/p6e8q1z1/ink-dev:latest "$@"
```

Comment:
IIRC, @fbielejec and @pmikolajczyk41 prefered to keep the version of the image pinned (here `0.1.0`) rather than using `latest`.

Reply:
Done